plugins/by-name/lsp-signature/default.nix

```
@@ -0,0 +1,208 @@
+      type = lib.types.anything;
+      pluginDefault = "🐼 ";
+      description = ''
+        Panda for parameter. Note: for the terminal not support emoji, might crash
+
+        or, provide a table with 3 icons
+      '';
+    };
+
+    hint_scheme = defaultNullOpts.mkStr "String" "";
+    hint_inline =
+      defaultNullOpts.mkLuaFn
+        ''
+          function() return false end
+        ''
+        ''
+          Should the hint be inline(nvim 0.10 only)?  default false
+          - Return true | 'inline' to show hint inline
+          - Return 'eol' to show hint at the end of line
+          - Return false to disable
+          - Return 'right_align' to display hint right aligned in the current line
+        '';
+
+    hi_parameter = defaultNullOpts.mkStr "LspSignatureActiveParameter" ''
+      How your parameter will be highlighted
+    '';
+
+    handler_opts =
+      defaultNullOpts.mkAttributeSet
+        {
+          border = "rounded";
+        }
+        ''
+          `border` can be double, rounded, single, shadow, none or a lua table of borders
+        '';
+
+    always_trigger = defaultNullOpts.mkBool false ''
+      Sometime show signature on new line or in middle of parameter can be confusing
+    '';
+
+    auto_close_after = defaultNullOpts.mkLua "nil" ''
+      Autoclose signature float win after x seconds, disabled if`nil`
+    '';
+
+    extra_trigger_chars = defaultNullOpts.mkListOf lib.types.str [ ] ''
+      Array of of extra characters that will trigger signature completion
+    '';
+
+    zindex = defaultNullOpts.mkInt 200 ''
+      By default it will be on top of all floating windows, set to <= 50 to send it to the bottom
+    '';
+
+    padding = defaultNullOpts.mkStr "" ''
+      Character to pad on left and right of signature
+    '';
+
+    transparency = defaultNullOpts.mkStrLuaOr (lib.types.ints.between 1 100) "nil" ''
+      Disabled by default, allow floating window transparent value 1~100
+    '';
+
+    shadow_blend = defaultNullOpts.mkInt 36 ''
+      If you're using shadow as border, use this to set the opacity
+    '';
+
+    shadow_guibg = defaultNullOpts.mkStr "Green" ''
+      If you're using shadow as border, use this to set the color
+    '';
+
+    time_interval = defaultNullOpts.mkInt 200 ''
+      Timer check interval. Set to a lower value if you want to reduce latency
+    '';
+
+    toggle_key = defaultNullOpts.mkLua "nil" ''
+      Toggle signature on and off in insert mode
+    '';
+
+    toggle_flip_floatwin_setting = defaultNullOpts.mkStrLuaOr lib.types.bool false ''
+      - true: toggle 
+      - floating_windows: true|false setting after toggle key pressed
+      - false: floating_windows setup will not change, toggle_key will pop up signature helper, but signature
+
+      May not popup when typing, depending on floating_window setting
+    '';
+
+    select_signature_key = defaultNullOpts.mkLua "nil" ''
+      The keybind for cycling to next signature
```

```suggestion
      The keybind for cycling to next signature.
```
Add a period at the end of each sentence. Sorry to bother you with this, but it helps ensuring consistency across our documentation.

---

will do